go.mod

```diff
@@ -50,6 +50,8 @@ require (
 	gotest.tools/v3 v3.5.0
 )
 
+require github.com/distribution/distribution/v3 v3.0.0-20230214150026-36d8c594d7aa
```

Comment:
See #10954 - will need to swap things to `github.com/distribution/reference` to keep this from reappearing.